lib/components/artist/artworks/index.js

```diff
@@ -113,26 +107,7 @@ export default Relay.createContainer(Artworks, {
           artworks
           for_sale_artworks
         }
-        for_sale_artworks: artworks_connection(sort: partner_updated_at_desc, filter: IS_NOT_FOR_SALE, first: $pageSize) {
-          edges {
-            node {
-              image {
-                id
-              }
-              ${Artwork.getFragment('artwork')}
-            }
-          }
-        }
-        not_for_sale_artworks: artworks_connection(sort: partner_updated_at_desc, filter: IS_NOT_FOR_SALE, first: $pageSize) {
-          edges {
-            node {
-              image {
-                id
-              }
-              ${Artwork.getFragment('artwork')}
-            }
-          }
-        }
+        ${ArtworksGrid.getFragment('artist')}
```

Comment:
Nope, we set the variables in the component constructor. And both grids are working now but Metaphysics is giving 'alternative facts' about `hasNextPage`, so the Other works grid renders too soon.